test/scripts/setup-konvoy.sh

```diff
@@ -10,4 +10,6 @@ fi
 
 set -euo pipefail
 
-curl --silent https://downloads.mesosphere.io/konvoy/konvoy_${KONVOY_VERSION}_${UNAME}.tar.bz2 | tar xjv --strip=1 konvoy_${KONVOY_VERSION}/konvoy
+if [[ ! -f konvoy ]]; then
```

Comment:
Could the existing Konvoy binary potentially be an old one? This could happen locally, would it get cached in CI?

Reply:
true, a version check would make sense here 👍

Reply:
At the moment, nothing would be cached or so. It is always downloaded into the workspace of the started job.

Reply:
How about the version check, just in case people run this locally (I sometimes do)?

Reply:
yes yes adding :)

Reply:
✔️